### PR TITLE
austenem/CAT-748 Fix support headers

### PR DESCRIPTION
**Pull request by austenem**
This PR fixes the Entity Headers for the detail pages of Support entities. Previously, these headers were blank for Support detail pages - they now contain relevant icons and ids, matching the production instance.

Screenshot 2024-07-11 at 12 12 28 PM
Screenshot 2024-07-11 at 12 13 08 PM

### CODE REVIEW

File: context/app/static/js/components/detailPage/entityHeader/EntityHeader/EntityHeader.tsx [hunk 1]
```diff
@@ -41,11 +42,7 @@ function Header() {
 
   return (
     <AnimatedPaper style={styles} elevation={4} data-testid="entity-header">
-      <EntityHeaderContent
-        assayMetadata={assayMetadata}
-        shouldDisplayHeader={shouldDisplayHeader}
-        vizIsFullscreen={vizIsFullscreen}
-      />
+      <EntityHeaderContent assayMetadata={assayMetadata} shouldDisplayHeader={shouldDisplayHeader} vizIsFullscreen />
```
Comment (NickAkhmetov) [inline context/app/static/js/components/detailPage/entityHeader/EntityHeader/EntityHeader.tsx:45]: If `vizIsFullscreen` is always set to true, the header always displays controls for a potentially non-existent vitessce visualization:
image


```suggestion
      <EntityHeaderContent assayMetadata={assayMetadata} shouldDisplayHeader={shouldDisplayHeader} vizIsFullscreen={vizIsFullscreen} />
```

With this suggestion:

image

**Review by NickAkhmetov — COMMENTED** [review]
Nice fix! I think the `vizIsFullscreen` change may have been unintentional?

File: context/app/static/js/components/detailPage/entityHeader/EntityHeader/EntityHeader.tsx (hunk 1 above)
Comment (austenem) [reply context/app/static/js/components/detailPage/entityHeader/EntityHeader/EntityHeader.tsx]: Ah good catch! That was an unintentional change from trying to resolve eslint errors.

**Review by NickAkhmetov — APPROVED** [review]
🙌🏻 Nicely done!

**Pull request merged**